app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuEnhancedTrackingProtectionRobot.kt

```diff
@@ -175,26 +176,26 @@ private fun assertEnhancedTrackingProtectionOptions() {
         .check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
 }
 
-private fun assertEnhancedTrackingProtectionOptionsGrayedOut() {
+private fun assertETPOptionsDisabled() {
```

Comment:
Probably better to make this function assert both scenarios as a general function by passing in a boolean so we don't have to write an 'Enabled' function in the future.

Reply:
I only renamed this, there are already 2 different functions written for what you are saying. I'll see what changes it implies to do this.

Reply:
@AaronMT don. please re-check.